lib/jet_plugin_sdk/api/graphql.ex

```diff
@@ -139,6 +140,7 @@ defmodule JetPluginSDK.API.GraphQL do
     quote location: :keep do
       object :jet_plugin_manifest do
         field :description, :string
+        field :endpoint, non_null(:string)
```

Comment:
这样是不是更好一些呀？不会和 lifecycle 混淆

```suggestion
        field :api_endpoint, non_null(:string)
```